app/services/html/prefix_all_images.rb

```diff
@@ -0,0 +1,36 @@
+module Html
+  class PrefixAllImages
+    def self.call(html, width = 880)
+      return unless html
```

Comment:
**nitpick (non-blocking):** Maybe `unless html.present?`, we don't need to do any work on empty strings.

```suggestion
      return unless html.present?
```